salt/states/powerdns.py

```diff
@@ -0,0 +1,118 @@
+"""
+Create, modify and delete PowerDNS zone
+"""
+
+import logging
+
+from salt.exceptions import SaltInvocationError
+
+log = logging.getLogger(__name__)
+
+
+def manage_zone(
```

Comment:
```suggestion
def managed_zone(
```

Reply:
Fixed

Reply:
> Fixed

Thanks! Could you please fix lint and pre-commit as well?